### PR TITLE
Add teamwork section to 'Managing a New Community and Team' chapter

### LINKED ISSUE #2207: More information on actual teamwork
### Summary

There is a lot of information about collaboration in the Turing Way, but not a lot on how to work in teams. 

We are developing an open-source course/guide: ['Teamwork for research software development'](https://nlesc.github.io/teamwork-for-research-software-development/), because we noticed there is a lot to learn about teamwork in a research (software) environment that you can not learn from the existing learning resources about teamwork frameworks (e.g. agile, scrum, kanban). It would be nice to add some information from the course and a reference it in the Turing way. I am happy to discuss where would be the best place to add this information.

Seems to be related to #1436, but that focuses only on task management (there is much more to teams than that).

The chapter https://the-turing-way.netlify.app/collaboration/new-community.html slightly touches the teamwork topic, but also doesn't really discuss it.

### What needs to be done?

- [ ] Decide on where to add new information about teamwork, I suggest a new chapter under https://the-turing-way.netlify.app/collaboration/collaboration.html called 'Working in a team'. But I am curious to hear other's opinions.
- [ ] Decide on what information about teamwork to add. I suggest to add a small executive summary of the ['Teamwork for research software development'](https://nlesc.github.io/teamwork-for-research-software-development/) course. And refer the reader to the course for further reading.

### Who can help?

* 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by svenvanderburg**
### Summary
Fixes #2207 

### List of changes proposed in this PR (pull-request)

- Add a new 'teamwork' section to 'Managing a New Community and Team' chapter
- Add link to teamwork section in 'communication & collaboration' part of project design

### What should a reviewer concentrate their feedback on?
- [ ] Everything looks ok?
- [ ] Is this the best place to put this?
- [ ] Is this the right level of detail? I think it's best to not repeat the whole lesson, but I am not sure.

### Acknowledging contributors

- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: `svenvanderburg`

### CODE REVIEW

**Comment by svenvanderburg** [comment]
@malvikasharan would you have time to take a look (see 'What should a reviewer concentrate their feedback on?' above for guidance)?

File: book/website/collaboration/new-community/new-community-teamwork.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# 
+(cl-new-community-teamwork)=
+# Teamwork
+The widespread standard in academia has been, and still is, to work individually for the majority of one’s time.
+However, if done well, working in a team can be more fun, more productive, and more effective.
+There are both benefits and downsides of working in teams, and the key is to work together in such a way that the advantages outweigh the disadvantages.
+
+There are many popular frameworks that provide useful guidance on how to best work in teams.
```
Comment (malvikasharan) [inline book/website/collaboration/new-community/new-community-teamwork.md:8]:
```suggestion
## Team Framework

There are many popular frameworks that provide useful guidance on how to best work in teams.
```

File: book/website/collaboration/new-community/new-community-teamwork.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# 
+(cl-new-community-teamwork)=
```
Comment (malvikasharan) [inline book/website/collaboration/new-community/new-community-teamwork.md:2]:
```suggestion
(cl-new-community-teamwork)=
```

**Comment by malvikasharan** [comment]
Thank you @svenvanderburg for the PR. I am back to work this week and hence the delay in review. 
The content looks good and I have added some minor formatting suggestion. There are some really useful [references from Altassian on teamwork](https://www.atlassian.com/blog/teamwork) that can be added at the end of the subchapter, for instance, [Master these 7 essential skills to level-up your teamwork game](https://www.atlassian.com/blog/teamwork/teamwork-skills-accelerate-career). What do you think?

Finally, to include this subchapter in the book, we need to mention the file path in the _toc.yml file: https://github.com/svenvanderburg/the-turing-way/blob/patch-1/book/website/_toc.yml, let me know if you'd like me to make this change or you're happy to add it there. I think this can easily with the first subchapter.

**Comment by svenvanderburg** [comment]
Thank you for your comments @malvikasharan!

## What I did in reaction to your comments:
- [x] Applied your formatting suggestions
- [x] Add reference to atlassian teamwork blog
- [x] Add filepath to toc
- [x] Add myself to contributors.

Can you take another look to see if this is OK?

File: book/website/_toc.yml [hunk 3]
```diff
@@ -415,6 +415,8 @@ parts:
         file: collaboration/new-community/new-community-techissues
       - title: Valuing Diversity and Differences
         file: collaboration/new-community/new-community-differences
+      - title: Guide to teamwork
```
Comment (malvikasharan) [inline book/website/_toc.yml:418]:
```suggestion
      - title: Teamwork
```

**Review by malvikasharan — APPROVED** [review]

**Comment by malvikasharan** [comment]
Thank you @svenvanderburg for the quick response. I have added additional references and also some images while I was at it 💪🏽. Thank you so much, the chapter looks great is really helpful at actually addressing the teamwork for explicitely. Approving this. Let me know when you are happy to have it merged.

**Comment by malvikasharan** [comment]
@all-contributors please add @svenvanderburg for content

**Comment by malvikasharan** [comment]
@svenvanderburg and @c-martinez, making a connection with the ongoing discussions on #2207, I have created a dedicated section in this chapter to highlight the tutorial. 

@svenvanderburg I am inviting you on this repo as a collaborator that should give you review and merge permission, when you are ready to merge this PR.

**Comment by svenvanderburg** [comment]
I only reverted the change of this PR in the Readme adding @svenvanderburg as contributor. This was done with #2250.

I like your additions @malvikasharan so I am merging!

**Pull request merged**